### PR TITLE
feat: accelerator fragment support cli change

### PR DESCRIPTION
**Pull request by guitarsheng**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/spring/azext_spring/_params.py [hunk 1]
```diff
@@ -1017,16 +1017,18 @@ def prepare_logs_argument(c):
...
-            c.argument('password', help='Password of git repository basic auth.')
-            c.argument('private_key', help='Private SSH Key algorithm of git repository.')
-            c.argument('host_key', help='Public SSH Key of git repository.')
-            c.argument('host_key_algorithm', help='SSH Key algorithm of git repository.')
+            c.argument('ca_cert_name', type=str, help='CA certificate name.')
+            c.argument('username', type=str, help='Username of git repository basic auth.')
+            c.argument('password', type=str, help='Password of git repository basic auth.')
+            c.argument('private_key', type=str, help='Private SSH Key algorithm of git repository.')
+            c.argument('host_key', type=str, help='Public SSH Key of git repository.')
+            c.argument('host_key_algorithm', type=str, help='SSH Key algorithm of git repository.')
```
Comment (zhoxing-ms) [inline src/spring/azext_spring/_params.py:1034]: `type=str` is the default value, why do you need to add them?

**Comment by zhoxing-ms** [comment]
Please add some tests for this PR change

File: src/spring/azext_spring/_params.py (hunk 1 above)
Comment (guitarsheng) [reply src/spring/azext_spring/_params.py:1034]: No special reason, should I remove it?

File: src/spring/azext_spring/_params.py [hunk 2]
```diff
@@ -1017,16 +1017,18 @@ def prepare_logs_argument(c):
             c.argument('description', type=str, help='Description for customized accelerator.')
             c.argument('icon_url', type=str, help='Icon url for customized accelerator.')
             c.argument('accelerator_tags', type=str, help="Comma-separated list of tags on the customized accelerator.")
+            c.argument('type', help='Type of customized accelerator.', arg_type=get_enum_type(CustomizedAcceleratorType))
 
-            c.argument('git_url', help='Git URL', validator=validate_acc_git_url)
+            c.argument('git_url', type=str, help='Git URL', validator=validate_acc_git_url)
+            c.argument('git_sub_path', type=str, help='Folder path inside the git repository to consider as the root of the accelerator or fragment.')
```
Comment (zhoxing-ms) [inline src/spring/azext_spring/_params.py:1023]: Same as above

File: src/spring/azext_spring/_params.py (hunk 1 above)
Comment (guitarsheng) [reply src/spring/azext_spring/_params.py]: Updated

**Review by zhoxing-ms — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**